### PR TITLE
[Redesign] Thanks

### PR DESCRIPTION
**Pull request by joelverhagen**
Address https://github.com/NuGet/NuGetGallery/issues/4271.

screencapture-localhost-account-thanks-1499119668800

### CODE REVIEW

File: src/NuGetGallery/Views/Users/Thanks.cshtml [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
     ViewBag.Title = "Thanks";
+    Layout = "~/Views/Shared/Gallery/Layout.cshtml";
 }
 
-<h1 class="page-heading">
-    Thank you for registering, @CurrentUser.Username!
-</h1>
+<article role="main" class="container main-container">
+    <div class="row">
+        <div class="col-xs-12 text-center">
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Users/Thanks.cshtml]: Nit: you could move the `text-center` class to the `row` div and not need two `text-center`s.

File: src/NuGetGallery/Views/Users/Thanks.cshtml [hunk 2]
```diff
@@ -1,28 +1,23 @@
...
-
-    <li>
-        <p>
-            <a href="@Url.UploadPackage()">Upload a package</a>.
-        </p>
-    </li>
-</ul>
+            <p>Make sure your email is verified. <a href="@Url.ConfirmationRequired()">Confirm your account</a></p>
+            
+            <p>Want to update your account? <a href="@Url.Action("Account", "Users")">View Account Settings</a></p>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Users/Thanks.cshtml]: I'm adding a `@Url.AccountSettings()` in the [drop down pull request](https://github.com/NuGet/NuGetGallery/pull/4328). Do you want to duplicate my changes, and then whoever merges in second removes the duplication?

Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/Thanks.cshtml]: sure

File: src/NuGetGallery/Views/Users/Thanks.cshtml (hunk 1 above)
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/Thanks.cshtml]: Good idea.

**Pull request merged**